ui-participant/src/studies/enroll/PreEnroll.tsx

```diff
@@ -61,7 +105,10 @@ export default function PreEnrollView({ enrollContext, survey }:
       studyEnvParams: { envName: studyEnv.environmentName, studyShortcode, portalShortcode },
       enrolleeShortcode: enrollee?.shortcode || '',
       referencedAnswers: [],
-      extraVariables: { isProxyEnrollment, isSubjectEnrollment, user }
+      extraVariables: {
+        isProxyEnrollment, isSubjectEnrollment, user,
+        eligibilityResult
```

Comment:
nit: should we do `eligible: eligibilityResult.eligible` so it's a bit shorter/easier to remember?